plugins/web/opentelemetry-instrumentation-long-task/src/instrumentation.ts

```diff
@@ -69,6 +77,9 @@ export class LongTaskInstrumentation extends InstrumentationBase {
     const span = this.tracer.startSpan(LONGTASK_PERFORMANCE_TYPE, {
       startTime: hrTime(entry.startTime),
     });
+    if (this._observerCallback) {
+      span.setAttributes(this._observerCallback(entry));
```

Comment:
Should we have protection for the case this callback throws? (try / catch)
This is an external callback that the instrumentation has no control of.
in node we usually wrap these kinds of calls with `safeExecuteInTheMiddle`

Reply:
Definitely safer with a try-catch. Added the feature and a test for that behavior